plugins/org.jboss.tools.jmx.remoting1_1_3/.classpath

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
No, the "main" way to load the client jars is to dynamically load them from an installation on disk. According to @robstryker this is done because not all client jars work with all versions of wildfly/eap. If this is only about supporting EAP images on openshift, I guess a single version (latest stable) should do and could be testable.